lib/internal/per_context.js

```diff
@@ -3,6 +3,16 @@
 // node::NewContext calls this script
 
 (function(global) {
+  // https://github.com/tc39/proposal-global
+  // https://github.com/nodejs/node/pull/22835
+  // TODO(devsnek,ljharb) remove when V8 71 lands
+  Object.defineProperty(global, 'globalThis', {
+    value: global,
+    writable: true,
+    enumerable: false,
+    configurable: true,
```

Comment:
would it be worthwhile emitting an experimental warning while this is still in early stages?

Reply:
I don’t see why; it’s not in early stages, it’s in stage 3, and v8 is shipping it unflagged.